InvenTree/InvenTree/middleware.py

```diff
@@ -71,10 +70,6 @@ def __call__(self, request):
 
             # No authorization was found for the request
             if not authorized:
-                # A logout request will redirect the user to the login screen
```

Comment:
Is this no longer required?

Reply:
@schrodingergat this will never trigger as all /account paths are excepted. And after logout you are always redirected to /index which sends you driectly to /account/login if you are not authenticated.